subjects/descendappendrange/README.md

```diff
@@ -0,0 +1,46 @@
+## descendappendrange
+
+### Instructions
+
+- Write a function that takes an `int` max and an `int` min as parameters. The function should return a slice of `int`s with all the values between the max and min.
+
+- The `max`must be included, and `min` must be excluded.
+
+- If `max` is inferior than or equal to `min`, return a `nil` value.
+
+- `make()` is not allowed for this exercise.
+
+### Expected function
+
+```go
+func DescendAppendRange(max, min int) []int {
+
+}
+```
+
+### Usage
+
+Here is a possible program to test your function:
+
+```go
+package main
+
+import (
+  "fmt"
+  "piscine"
+)
+
+func main() {
+  fmt.Println(piscine.DescendAppendRange(10, 5))
+  fmt.Println(piscine.DescendAppendRange(5, 10))
+}
+```
+
+And its output:
+
+```console
+$ go run .
+[10 9 8 7 6]
+[]
```

Comment:
Shouldn't it return nil?

Reply:
Appears the purpose is to really return an empty array

Reply:
Your solution returns nil, if you tell the learner to return an empty slice, when testing it it would fail.

Reply:
Isn't `[]` and empty slice?